### PR TITLE
Add ganon triforce hunt

### PR DESCRIPTION
**Pull request by CaitSith2**

### CODE REVIEW

**Comment by CaitSith2** [comment]
Obsoleted by PR #117

File: EntranceRandomizer.py [hunk 1]
```diff
@@ -79,7 +79,11 @@ def defval(value):
                              Triforce Hunt: Places 30 Triforce Pieces in the world, collect
                                             20 of them to beat the game.
                              Local Triforce Hunt: Places 30 Triforce Pieces in your world, collect
-                                            20 of them to beat the game.       
+                                            20 of them to beat the game.
+                             Ganon Triforce Hunt: Places 30 Triforce Pieces in the world, collect
+                                            20 of them, then defeat Ganon.
+                             Local Ganon Triforce Hunt: Places 30 Triforce Pieces in the world,
```
Comment (Berserker66) [inline EntranceRandomizer.py:85]: local triforce hunt says "in your world", this wasn't copied to local ganon triforce hunt

File: Fill.py [hunk 2]
```diff
@@ -244,8 +244,8 @@ def distribute_items_restrictive(world, gftower_trash=False, fill_locations=None
             continue
 
         gftower_trash_count = (
-            random.randint(15, 50) if world.goal[player] in {'triforcehunt', 'localtriforcehunt'} else random.randint(0,
-                                                                                                                      15))
+            random.randint(15, 50) if world.goal[player] in {'triforcehunt', 'localtriforcehunt', 'ganontriforcehunt', 'localganontriforcehunt'}
```
Comment (Berserker66) [inline Fill.py:247]: I think we're getting to the point where we want if 'triforcehunt' in world.goal[player]

File: EntranceRandomizer.py (hunk 1 above)
Comment (CaitSith2) [reply EntranceRandomizer.py]: Resolved in d843119

File: Fill.py (hunk 2 above)
Comment (CaitSith2) [reply Fill.py]: Resolved in f9c40a8

**Pull request merged**